Description: Convert mirage.png to mirage.xpm for a menu icon
Author: Thomas Ross <th0m4sr0ss@gmail.com>
Last-Update: 2015-07-30
--- /dev/null
+++ b/mirage.xpm
@@ -0,0 +1,329 @@
+/* XPM */
+static char * mirage_xpm[] = {
+"128 128 198 2",
+"  	c None",
+". 	c #377500",
+"+ 	c #387700",
+"@ 	c #3B7C01",
+"# 	c #3F8202",
+"$ 	c #428702",
+"% 	c #458C03",
+"& 	c #468E04",
+"* 	c #478F04",
+"= 	c #489104",
+"- 	c #499304",
+"; 	c #4B9505",
+"> 	c #4C9705",
+", 	c #4B9605",
+"' 	c #4A9304",
+") 	c #489004",
+"! 	c #468D04",
+"~ 	c #448B03",
+"{ 	c #428803",
+"] 	c #418502",
+"^ 	c #3D7F01",
+"/ 	c #397900",
+"( 	c #387600",
+"_ 	c #4E9A06",
+": 	c #408302",
+"< 	c #3A7B00",
+"[ 	c #4A9405",
+"} 	c #4D9905",
+"| 	c #3C7D01",
+"1 	c #377600",
+"2 	c #3E8101",
+"3 	c #4D9805",
+"4 	c #428703",
+"5 	c #3A7A00",
+"6 	c #4A9505",
+"7 	c #3B7B01",
+"8 	c #3F8302",
+"9 	c #3E8102",
+"0 	c #3E8001",
+"a 	c #499204",
+"b 	c #3A7B01",
+"c 	c #397800",
+"d 	c #478E04",
+"e 	c #438803",
+"f 	c #428602",
+"g 	c #438903",
+"h 	c #3A7900",
+"i 	c #387800",
+"j 	c #3C7E01",
+"k 	c #458B03",
+"l 	c #4C9805",
+"m 	c #4C9605",
+"n 	c #468D03",
+"o 	c #408402",
+"p 	c #448A03",
+"q 	c #3D8001",
+"r 	c #397400",
+"s 	c #637300",
+"t 	c #747300",
+"u 	c #507400",
+"v 	c #607500",
+"w 	c #B17600",
+"x 	c #C37700",
+"y 	c #AB7600",
+"z 	c #4A7500",
+"A 	c #727500",
+"B 	c #C17600",
+"C 	c #B67600",
+"D 	c #517500",
+"E 	c #387500",
+"F 	c #847600",
+"G 	c #BB7600",
+"H 	c #507500",
+"I 	c #377400",
+"J 	c #917600",
+"K 	c #AA7600",
+"L 	c #3A7500",
+"M 	c #458D03",
+"N 	c #726D00",
+"O 	c #827600",
+"P 	c #586900",
+"Q 	c #A96600",
+"R 	c #587500",
+"S 	c #4D6D00",
+"T 	c #8F5600",
+"U 	c #B07600",
+"V 	c #3C7400",
+"W 	c #8B5800",
+"X 	c #915600",
+"Y 	c #7B6B00",
+"Z 	c #3D7E01",
+"` 	c #A86500",
+" .	c #516B00",
+"..	c #885800",
+"+.	c #3E7200",
+"@.	c #706100",
+"#.	c #387400",
+"$.	c #418602",
+"%.	c #C9B400",
+"&.	c #C5AE00",
+"*.	c #C7B100",
+"=.	c #CCB704",
+"-.	c #D2BD0E",
+";.	c #D8C317",
+">.	c #DECA21",
+",.	c #E4D02A",
+"'.	c #E8D430",
+").	c #EAD633",
+"!.	c #ECD836",
+"~.	c #E8D135",
+"{.	c #E1CC28",
+"].	c #C9B401",
+"^.	c #DAC61B",
+"/.	c #E0CC24",
+"(.	c #E7D32E",
+"_.	c #EDD937",
+":.	c #F3DF41",
+"<.	c #F9E64A",
+"[.	c #FCE94F",
+"}.	c #F5DF4A",
+"|.	c #F8E44C",
+"1.	c #D0BB0B",
+"2.	c #DBC61B",
+"3.	c #E5D12C",
+"4.	c #F0DC3D",
+"5.	c #FAE74C",
+"6.	c #CDB806",
+"7.	c #D9C419",
+"8.	c #E3CF29",
+"9.	c #EEDA39",
+"0.	c #F8E54A",
+"a.	c #D0BB0A",
+"b.	c #E0CB23",
+"c.	c #F0DC3C",
+"d.	c #FBE84E",
+"e.	c #D3BE0F",
+"f.	c #E3CF28",
+"g.	c #D2BD0D",
+"h.	c #E6D22D",
+"i.	c #F6E246",
+"j.	c #D5C013",
+"k.	c #EDD938",
+"l.	c #935700",
+"m.	c #975A00",
+"n.	c #D9C418",
+"o.	c #F0DD3D",
+"p.	c #F9E44C",
+"q.	c #965E04",
+"r.	c #955C03",
+"s.	c #FAE74E",
+"t.	c #CAB501",
+"u.	c #DCC81E",
+"v.	c #F4E042",
+"w.	c #D9B935",
+"x.	c #B6891B",
+"y.	c #B5881A",
+"z.	c #B78A1C",
+"A.	c #DEC039",
+"B.	c #F1DE3E",
+"C.	c #DDC91F",
+"D.	c #FAE64B",
+"E.	c #D5C113",
+"F.	c #F7E447",
+"G.	c #D0BC0C",
+"H.	c #F9E54A",
+"I.	c #F4E143",
+"J.	c #EDDA38",
+"K.	c #E2CE27",
+"L.	c #CAB502",
+"M.	c #F8E549",
+"N.	c #D3BF10",
+"O.	c #DFCB22",
+"P.	c #DFCA22",
+"Q.	c #DECA20",
+"R.	c #DEC920",
+"S.	c #DDC920",
+"T.	c #DDC81F",
+"U.	c #DDC81E",
+"V.	c #DCC71D",
+"W.	c #DBC71D",
+"X.	c #DBC71C",
+"Y.	c #DBC61C",
+"Z.	c #DAC61A",
+"`.	c #DAC51A",
+" +	c #D9C51A",
+".+	c #D9C519",
+"++	c #D8C418",
+"@+	c #D8C417",
+"#+	c #D7C316",
+"$+	c #D7C216",
+"%+	c #D7C215",
+"&+	c #D6C215",
+"*+	c #D6C114",
+"=+	c #D5C012",
+"-+	c #D4C012",
+";+	c #D4C011",
+">+	c #D4BF11",
+",+	c #CEB907",
+"                                                                                                                                                                                                                                                                ",
+"                                                                                                                                                                                                                                                                ",
+"                                                                                                                            . . .                                                                                                                               ",
+"                                                                                                        . . . . . . . . . . . . . . . . . . . . . . . . .                                                                                                       ",
+"                                                                                              . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . .                                                                                               ",
+"                                                                                        . . . . . . . . . + @ # $ % & * = - ; > , ' ) ! ~ { ] # ^ @ / ( . . . . . . . . .                                                                                       ",
+"                                                                                  . . . . . . . + ^ { ) > _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ ; % : < . . . . . . . . .                                                                               ",
+"                                                                            . . . . . . . ^ ~ [ } _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ > & ] | . . . . . . .                                                                           ",
+"                                                                        . . . . . 1 2 % > _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 3 ) 4 5 . . . . . .                                                                     ",
+"                                                                    . . . . . @ % 3 _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ } * ^ . . . . . .                                                                 ",
+"                                                                . . . . . # = _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 6 ] + . . . . .                                                             ",
+"                                                            . . . . 1 ] , _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 3 ~ 7 . . . . .                                                         ",
+"                                                        . . . . . 8 ; _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ = 9 . . . .                                                       ",
+"                                                      . . . . 0 [ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ a b . . . .                                                   ",
+"                                                  . . . . c d _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ } $ . . . .                                                 ",
+"                                                . . . . 8 > _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ - @ . . . .                                             ",
+"                                              . . . / * _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ } e 1 . . .                                           ",
+"                                          . . . . ^ > _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ [ | . . .                                         ",
+"                                        . . . . f } _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ } g . . . .                                     ",
+"                                      . . . + & _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ d 1 . . .                                   ",
+"                                    . . . c ' _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ a c . . .                                 ",
+"                                  . . . c [ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ ; 5 . . .                               ",
+"                                . . . / 6 _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ ; / . . .                             ",
+"                              . . . h ; _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ [ i . . .                           ",
+"                              . . + [ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ a ( . .                           ",
+"                            . . . = _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ * . . .                         ",
+"                          . . . % _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ $ . . .                       ",
+"                        . . . f _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ } j . . .                     ",
+"                        . . | } _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ , i . .                     ",
+"                      . . ( 6 _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ * . . .                   ",
+"                    . . . k _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 0 . .                   ",
+"                    . . 0 _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ } g 9 { - _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 6 1 . .                 ",
+"                  . . 1 ; _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ } _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 9 . . . . 7 * _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ f . . .               ",
+"                  . . ] _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ ) ^ . / = _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ ) . . . . . . . ^ ; _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 3 c . .               ",
+"                . . + > _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ } ] . . . . ( ; _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ ] . .     . . . . c a _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 4 . .               ",
+"              . . . g _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ l ^ . . . . . . 8 _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 7 . .         . . . 1 ! _ _ _ _ _ _ _ _ _ _ _ _ _ _ , . . .             ",
+"              . . c 3 _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ ; b . . .     . . + } _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ m . . .           . . . . { _ _ _ _ _ _ _ _ _ _ _ _ _ _ # . .             ",
+"              . . 4 _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ a + . . .       . . . a _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ n . .                 . . . # 3 _ _ _ _ _ _ _ _ _ _ _ _ a . . .           ",
+"            . . . , _ _ _ _ _ _ _ _ _ _ _ _ _ _ = + . . .           . . g _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 4 . .                   . . . 2 } _ _ _ _ _ _ _ _ _ _ _ _ 7 . .           ",
+"            . . 9 _ _ _ _ _ _ _ _ _ _ _ _ _ _ * 1 . . .             . . 0 _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ o . .                     . . . ] _ _ _ _ _ _ _ _ _ _ _ _ f . .           ",
+"          . . . = _ _ _ _ _ _ _ _ _ _ _ _ _ n . . . .               . . 7 _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 0 . .                       . . . p _ _ _ _ _ _ _ _ _ _ _ a . . .         ",
+"          . . / _ _ _ _ _ _ _ _ _ _ _ _ _ ~ . . . .                 . . / _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ m a 3 _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ @ . .                         . . . * _ _ _ _ _ _ _ _ _ _ } c . .         ",
+"          . . o _ _ _ _ _ _ _ _ _ _ _ _ g . . .                     . . + _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ } p 5 . . 1 2 ! } _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ / . .                           . . ( a _ _ _ _ _ _ _ _ _ _ : . .         ",
+"        . . . * _ _ _ _ _ _ _ _ _ _ _ $ . . .                       . . . } _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ [ 5 . . . . . . . q 3 _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ } . . .                           . . . c ; _ _ _ _ _ _ _ _ _ k . .         ",
+"        . . ( } _ _ _ _ _ _ _ _ _ } o . . .                         . . . > _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ = + . . r s t u . . . | > _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ m . . .                             . . . 7 > _ _ _ _ _ _ _ _ a . . .       ",
+"        . . 2 _ _ _ _ _ _ _ _ _ } # . . .                           . . . ; _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ % . . . v w x x x y z . . 5 m _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ - . .                                 . . . ^ } _ _ _ _ _ _ _ 3 . . .       ",
+"        . . p _ _ _ _ _ _ _ _ 3 0 . . .                               . . - _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ p . . . A B x x x x x C D . . q _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ * . .                                   . . . 8 _ _ _ _ _ _ _ _ 5 . .       ",
+"        . . ) _ _ _ _ _ _ _ > | . . .                                 . . ) _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ * . . E F x x x x x x x x G H . . ~ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ k . .                                     . . . { _ _ _ _ _ _ _ 0 . .       ",
+"      . . . m _ _ _ _ _ _ [ 5 . . .                                   . . & _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ - ( . I J x x x x x x x x x x K L . ( 6 _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ { . .                                       . . . M _ _ _ _ _ _ : . .       ",
+"      . . i _ _ _ _ _ _ = + . . .                                     . . k _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ , c . . N x x x x x x x x x x x x O . . | } _ _ _ _ _ _ _ _ _ _ _ _ _ _ o . .                                         . . 1 = _ _ _ _ _ ] . .       ",
+"      . . j _ _ _ _ _ & 1 . . .                                       . . g _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 3 7 . . P Q x x x x x x x x x x x x B R . . e _ _ _ _ _ _ _ _ _ _ _ _ _ _ 0 . .                                           . . i 6 _ _ _ _ { . .       ",
+"      . . # _ _ _ _ p . . . .                                         . . $ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 8 . . S T Q x x x x x x x x x x x x x U V . . - _ _ _ _ _ _ _ _ _ _ _ _ _ @ . .                                           . . . 5 > _ _ _ p . .       ",
+"      . . o _ _ } ] . . .                                             . . ] _ _ _ _ _ _ _ _ _ _ _ _ _ _ M . . I W X Q x x x x x x x x x x x x x x Y . . 7 } _ _ _ _ _ _ _ _ _ _ _ _ c . .                                             . . . Z } _ _ % . .       ",
+"      . . f _ 3 9 . . .                                               . . # _ _ _ _ _ _ _ _ _ _ _ _ _ [ ( . .   X X Q x x x x x x x x x x x x x x `  .. . f _ _ _ _ _ _ _ _ _ _ _ } . . .                                               . . . q 3 _ k . .       ",
+"      . . g - @ . . .                                                 . . 0 _ _ _ _ _ _ _ _ _ _ _ _ } @ . .     X X Q x x x x x x x x x x x x x x ` ..+.. . ' _ _ _ _ _ _ _ _ _ _ m . . .                                                 . . . @ m e . .       ",
+"      . . b . . . .                                                   . . j _ _ _ _ _ _ _ _ _ _ _ _ o . . .     X X Q x x x x x x x x x x x x x x ` X @.. . | } _ _ _ _ _ _ _ _ _ - . . .                                                   . . . h @ . .       ",
+"      . . . . . .                                                     . . @ _ _ _ _ _ _ _ _ _ _ _ * . . .       X X Q x x x x x x x x x x x x x x ` X X #.. . p _ _ _ _ _ _ _ _ _ * . .                                                       . . . . . .       ",
+"      . . . .                                                         . . c _ _ _ _ _ _ _ _ _ _ , i . .         X X Q x x x x x x x x x x x x x x ` X X   . . + , _ _ _ _ _ _ _ _ k . .                                                         . . . .         ",
+"                                                                      . . . 3 _ _ _ _ _ _ _ _ } ^ . . .         X X Q x x x x x x x x x x x x x x ` X X     . . 2 _ _ _ _ _ _ _ _ ] . .                                                                         ",
+"                                                                        . . ' _ _ _ _ _ _ _ _ g . . .           X X Q x x x x x x x x x x x x x x ` X X     . . . d _ _ _ _ _ _ _ q . .                                                                         ",
+"                                                                        . . & _ _ _ _ _ _ _ a . . .             X X Q x x x x x x x x x x x x x x ` X X       . . / l _ _ _ _ _ _ 5 . .                                                                         ",
+"                                                                        . . g _ _ _ _ _ _ 3 5 . .               X X Q x x x x x x x x x x x x x x ` X X       . . . ] _ _ _ _ _ 3 . . .                                                                         ",
+"                                                                        . . | _ _ _ _ _ _ 8 . . .               X X Q x x x x x x x x x x x x x x ` X X         . . . n _ _ _ _ - . . .                                                                         ",
+"                                                                        . . . ; _ _ _ _ n . . .                 X X Q x x x x x x x x x x x x x x ` X X           . . 1 - _ _ _ $.. .                                                                           ",
+"                                                                          . . g _ _ _ - + . .                   X X Q x x x x x x x x x x x x x x ` X X             . . / ; _ } c . .                                                                           ",
+"                                                                          . . 5 } _ [ i . . .                   X X Q x x x x x x x x x x x x x x ` X X             . . . + n * . . .                                                                           ",
+"                                                                          . . . # ' / . . .                     X X Q x x x x x x x x x x x x x x ` X X               . . . . 1 . .                                                                             ",
+"                                                                            . . . . . . .                       X X Q x x x x x x x x x x x x x x ` X X                 . . . . . .                                                                             ",
+"                                                                              . . . . .                         X X Q x x x x x x x x x x x x x x ` X X                     . .                                                                                 ",
+"                                                                                . .                             X X Q x x x x x x x x x x x x x x ` X X                                                                                                         ",
+"                                                                                                                X X Q x x x x x x x x x x x x x x ` X X                                                                                                         ",
+"                                                                                                                X X Q x x x x x x x x x x x x x x ` X X                                                                                                         ",
+"                                                                                                                X X Q x x x x x x x x x x x x x x ` X X                                                                                                         ",
+"                                                                                                                X X Q x x x x x x x x x x x x x x ` X X                                                                                                         ",
+"                                                                                                                X X Q x x x x x x x x x x x x x x ` X X                                                                                                         ",
+"                                                                                                                X X Q x x x x x x x x x x x x x x ` X X                                                                                                         ",
+"                                                                                                                X X Q x x x x x x x x x x x x x x ` X X                                                                                                         ",
+"                                                                                                                X X Q x x x x x x x x x x x x x x ` X X                                                                                                         ",
+"                                                                                                                X X Q x x x x x x x x x x x x x x ` X X                                                                                                         ",
+"                                                                                                                X X Q x x x x x x x x x x x x x x ` X X                                                                                                         ",
+"                                                                                                                X X Q x x x x x x x x x x x x x x ` X X                                                                                                         ",
+"                                                                                                                X X Q x x x x x x x x x x x x x x ` X X                                                                                                         ",
+"                                                                                                                X X Q x x x x x x x x x x x x x x ` X X                                                                                                         ",
+"                                                                                                                X X Q x x x x x x x x x x x x x x ` X X                                                                                                         ",
+"                                                                                                                X X Q x x x x x x x x x x x x x x ` X X                                                                                                         ",
+"                                                                                                                X X Q x x x x x x x x x x x x x x ` X X                                                                                                         ",
+"                                                                                                                X X Q x x x x x x x x x x x x x x ` X X                                                                                                         ",
+"                                                                                                                X X Q x x x x x x x x x x x x x x ` X X                                                                                                         ",
+"                                                                                                                X X Q x x x x x x x x x x x x x x ` X X                                                                                                         ",
+"                                                                                                                X X Q x x x x x x x x x x x x x x ` X X                                                                                                         ",
+"                                                                                                                X X Q x x x x x x x x x x x x x x ` X X                                                                                                         ",
+"                                                                                                                X X Q x x x x x x x x x x x x x x ` X X                                                                                                         ",
+"                                                                                                                X X Q x x x x x x x x x x x x x x ` X X                                                                                                         ",
+"                                                                                                                X X Q x x x x x x x x x x x x x x ` X X                                                                                                         ",
+"                                                                                                                X X Q x x x x x x x x x x x x x x ` X X                                                                                                         ",
+"                                                                                                                X X Q x x x x x x x x x x x x x x ` X X                                                                                                         ",
+"                                                                                                                X X Q x x x x x x x x x x x x x x ` X X                                                                                                         ",
+"                                                                                                                X X Q x x x x x x x x x x x x x x ` X X                                                                                                         ",
+"                                                                                                                X X Q x x x x x x x x x x x x x x ` X X                                                                                                         ",
+"                                                                                                                X X Q x x x x x x x x x x x x x x ` X X                                                                                                         ",
+"                                                                                                                X X Q x x x x x x x x x x x x x x ` X X                                                                                                         ",
+"                                                                                                                X X Q x x x x x x x x x x x x x x ` X X                                                                                                         ",
+"                                                                                                                X X Q x x x x x x x x x x x x x x ` X X                                                                                                         ",
+"                                                                                          %.%.%.%.%.%.%.%.%.%.&.X X Q x x x x x x x x x x x x x x ` X X *.%.%.%.%.%.%.                                                                                          ",
+"                                                                            %.%.%.%.%.%.%.%.%.%.%.%.%.%.%.%.%.&.X X Q x x x x x x x x x x x x x x ` X X *.%.%.%.%.%.%.%.%.%.%.%.%.%.                                                                            ",
+"                                                                    %.%.%.%.%.%.%.%.%.%.%.%.%.=.-.;.>.,.'.).!.~.X X Q x x x x x x x x x x x x x x ` X X {.>.;.-.=.%.%.%.%.%.%.%.%.%.%.%.%.%.                                                                    ",
+"                                                          %.%.%.%.%.%.%.%.%.%.].-.^./.(._.:.<.[.[.[.[.[.[.[.[.}.X X Q x x x x x x x x x x x x x x ` X X |.[.[.[.[.<.:._.(./.^.-.].%.%.%.%.%.%.%.%.%.%.                                                          ",
+"                                                    %.%.%.%.%.%.%.%.%.1.2.3.4.5.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.}.X X Q x x x x x x x x x x x x x x ` X X |.[.[.[.[.[.[.[.[.[.[.[.5.4.3.2.1.%.%.%.%.%.%.%.%.%.                                                    ",
+"                                              %.%.%.%.%.%.%.6.7.8.9.0.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.}.X X Q x x x x x x x x x x x x x x ` X X |.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.0.9.8.7.6.%.%.%.%.%.%.%.                                              ",
+"                                        %.%.%.%.%.%.%.a.b.c.d.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.}.X X Q x x x x x x x x x x x x x x ` X X |.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.d.c.b.a.%.%.%.%.%.%.%.                                        ",
+"                                    %.%.%.%.%.%.e.f.:.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.}.X X Q x x x x x x x x x x x x x x ` X X |.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.:.f.e.%.%.%.%.%.%.                                    ",
+"                                %.%.%.%.%.g.h.i.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.}.X X Q x x x x x x x x x x x x x x ` X X |.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.i.h.g.%.%.%.%.%.                                ",
+"                            %.%.%.%.%.j.k.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.}.X X l.m.m.m.m.m.m.m.m.m.m.m.m.m.m.l.X X |.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.k.j.%.%.%.%.%.                            ",
+"                        %.%.%.%.%.n.o.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.p.q.X X X X X X X X X X X X X X X X X X r.s.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.o.n.%.%.%.%.%.                        ",
+"                      %.%.%.t.u.v.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.w.x.y.y.y.y.y.y.y.y.y.y.y.y.y.y.y.y.z.A.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.v.u.t.%.%.%.                      ",
+"                  %.%.%.%.1.B.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.B.1.%.%.%.%.                  ",
+"                %.%.%.%.C.D.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.D.C.%.%.%.%.                ",
+"              %.%.%.6.!.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.!.6.%.%.%.              ",
+"            %.%.%.E.F.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.F.E.%.%.%.            ",
+"          %.%.%.G.H.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.H.G.%.%.%.          ",
+"          %.%.=.I.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.I.=.%.%.          ",
+"        %.%.%.J.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.J.%.%.%.        ",
+"        %.%.K.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.K.%.%.        ",
+"      %.%.L.M.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.M.L.%.%.      ",
+"      %.%.^.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.[.^.%.%.      ",
+"      %.%.N.O.P.>.>.>.Q.R.S.C.C.T.U.u.u.u.V.V.W.X.X.Y.2.^.^.^.Z.`.`. +.+.+7.n.++++@+;.;.#+#+$+%+%+&+&+*+*+*+*+E.E.j.=+=+-+;+>+>+>+>+>+>+>+>+>+>+>+>+>+>+>+>+>+>+>+>+>+>+>+>+>+>+>+>+>+>+>+>+>+>+>+>+>+>+>+>+>+>+>+>+>+>+>+>+>+>+>+>+>+>+>+>+>+>+>+>+,+%.%.      ",
+"      %.%.%.%.%.%.%.%.%.%.%.%.%.%.%.%.%.%.%.%.%.%.%.%.%.%.%.%.%.%.%.%.%.%.%.%.%.%.%.%.%.%.%.%.%.%.%.%.%.%.%.%.%.%.%.%.%.%.%.%.%.%.%.%.%.%.%.%.%.%.%.%.%.%.%.%.%.%.%.%.%.%.%.%.%.%.%.%.%.%.%.%.%.%.%.%.%.%.%.%.%.%.%.%.%.%.%.%.%.%.%.%.%.%.%.%.%.%.%.%.%.%.      ",
+"      %.%.%.%.%.%.%.%.%.%.%.%.%.%.%.%.%.%.%.%.%.%.%.%.%.%.%.%.%.%.%.%.%.%.%.%.%.%.%.%.%.%.%.%.%.%.%.%.%.%.%.%.%.%.%.%.%.%.%.%.%.%.%.%.%.%.%.%.%.%.%.%.%.%.%.%.%.%.%.%.%.%.%.%.%.%.%.%.%.%.%.%.%.%.%.%.%.%.%.%.%.%.%.%.%.%.%.%.%.%.%.%.%.%.%.%.%.%.%.%.%.%.      "};
